app/views/early_years_payment/provider/authenticated/claims/_ineligibility_returner.html.erb

```diff
@@ -0,0 +1,32 @@
+<% first_name = journey_session.answers.first_name %>
+
+<% content_for(:page_title, page_title("#{first_name} is not eligible for this payment", journey: current_journey_routing_name)) %>
+<% @backlink_path = claim_path("early-years-payment", "landing-page") %>
```

Comment:
Is this backlink correct?
On the FE journey, the backlink on ineligibility screens actually goes back to the question just asked (same as hitting the back button).